lib/Manager/TokenManager.php

```diff
@@ -163,6 +176,12 @@ public function getToken($videoManagerId = null)
         $logger = $this->getLogger();
         $this->logTokenData();
 
+        $cacheKey = sha1(sprintf('%s.%s', __METHOD__, json_encode(func_get_args())));
```

Comment:
I prefer to not to use sprintf if it doesn't help, and the "." looks useless. Sprintf is about 2-3 times slower in comparison to concat in PHP7.1 (PHP7.x is around 5 times faster than PHP5.6 💛 )

https://3v4l.org/nqJ6m

PS: because it's not a problem and just a suggestion, I'll approve it. It's upon you.

Reply:
@omid I sincerely doubt that we or other consumers will end up on a situation where these 2ms matter when building a PHP application

Reply:
You're right @omid, the `.` is totally useless. But I'll just leave it as it is, cause it makes no difference really. As for sprintf, there was a policy in my previous company to always use sprintf instead of concatenation, so it just got under my skin...